homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  width: 1280px;
+  margin: 0 auto;
+  overflow-x: hidden;
+}
+
+* {
+  font-family: Roboto, sans-serif;
+}
+
+/*
+.header {
+    background: #EF4934;
+    width: 100vw;
+    height: 80px;
+}
+*/
+
+.about-me {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: space-between;
+  margin: 53px 52px 0 53px;
+  gap: 68px;
+}
+
+.img-about {
+  width: 50%;
+  max-width: none;
+  display: block;
+  margin: 0 auto;
+}
+
+img {
+  width: 100%;
+}
+
+.text-about {
+  width: 50%;
+  max-width: none;
+  display: block;
+}
+
+.text-about>*+* {
```

Comment:
This selector is hard to read and understand. Do you need it? Maybe better to add this margin to paragraphs?

Reply:
I fixed it by adding margin-top